lib/compileTemplate.ts

```diff
@@ -40,6 +40,16 @@ export function compileTemplate (
     return actuallyCompile(Object.assign({}, options, {
       source: preprocess(options, preprocessor)
     }))
+  } else if (preprocessLang) {
+    return {
+      code: (
+        `var render = function () {}\n` +
+        `var staticRenderFns = []\n`
+      ),
+      source: options.source,
+      tips: [`Please install ${preprocessLang}`],
+      errors: [`Please install ${preprocessLang}`]
```

Comment:
The error message should tell what is not working.
And then the tip can suggest how to fix it.

Reply:
Updated, wanted to make sure I had my logic correct first